paper/paper.md

```diff
@@ -0,0 +1,75 @@
+PreliZ distributions are built on top of SciPy distributions, providing easy access to all standard functionality, in particular random sampling and quantiles needed in the elicitation process. However, PreliZ extends many of the distributions for increased user-friendliness. For instance, PreliZ uses a canonical parametrization, instead of loc-scale style used by many of the SciPy distributions, and adds common alternative parametrizations,  e.g. Beta-distribution also in terms of mean and standard deviation. Finally, PreliZ also includes some distributions not available in SciPy, like the HalfStudent, or LogitNormal.
+
+Preliz provides a range of plotting commands, including probability densities and cumulative densities ('plot_pdf' and 'plot_cdf') as well as extensive numerical summaries covering e.g. equal tailed intervals and highest density intervals. All plots support easy integration of input functionality, with the method 'plot_interactive' automatically creating ipywidgets sliders for effortless exploration of how the parameters affect the distribution (see \autoref{fig:figure_1}).
+
+![Figure 1: Example of plot_interactive() output for Gamma distribution with mu=2 and sigma=1.\label{fig:figure_1}](figures/figure_1.png)
+
+PreliZ includes a few algorithms for inferring the suitable prior from the user-provided information, and provides a general API that makes adding new algorithms easy. As a practical example, 'pz.maxent(pz.Gamma(mu=4), 1, 10, 0.9)' would find the maximum-entropy Gamma distribution that has 90% of its mass between 1 and 10, with the additional constraint of mean being 4. The library currently implements also the roulette method of '@morris:2014', allowing users to find a prior distribution by drawing a histogram (see \autoref{fig:figure_2}).
+
+![Figure 2: Roulette method.\label{fig:figure_2}](figures/figure_2.png)
+
+One of the main goals of PreliZ is to facilitate predictive elicitation, where information provided in the space of observable quantities is converted into a valid prior by a suitable learning algorithms, avoiding the need to provide direct information on the model parameters. PreliZ already implements functionality for this. For instance, 'pz.predictive_sliders` (see \autoref{fig:figure_3}) makes it easy to explore how the prior predictive distribution, of an arbitrary probabilistic model, changes with the prior, and `pz.ppa` (see figure XXX) is an experimental prior predictive assistant that computes priors consistent with user-made choice of prior predictive samples. 
+
+![Figure 3: Example of use of predictive_sliders for a simple model.\label{fig:figure_3}](figures/figure_3.png)
+
+# Software Citations
+
+PreliZ is written in Python 3.8+ and uses the following software packages:
+
+* ArviZ [@kumar:2019]
+* ipympl
+* ipywidgets 
```

Comment:
This seems to be the official citation 

@unpublished{interactive_Jupyter_widgets,
author = {Jupyter widgets community},
title = {ipywidgets, a GitHub repository},
year = {2015},
note = {Retrieved from https://github.com/jupyter-widgets/ipywidgets}
}